docs/getting-started/compose-infrastructure.md

```diff
@@ -1,37 +1,35 @@
 ---
-title: Publish Infrastructure
+title: Compose Infrastructure
 toc: true
 weight: 4
 indent: true
 ---
 
-# Publish Infrastructure
-
-Provisioning infrastructure using the Kubernetes API is a powerful capability,
-but combining primitive infrastructure resources into a single unit and
-publishing them to be provisioned by developers and consumed by applications
-greatly enhances this functionality.
-
-As mentioned in the [last section], CRDs that represent infrastructure resources
-on a provider are installed at the *cluster scope*. However, applications are
-typically provisioned at the *namespace scope* using Kubernetes primitives such
-as `Pod` or `Deployment`. To make infrastructure resources available to be
-provisioned at the namespace scope, they can be *published*. This consists of
-creating the following resources:
-
-- `InfrastructureDefinition`: defines a new kind of composite resource
-- `InfrastructurePublication`: makes an `InfrastructureDefinition` available at
-  the namespace scope
-- `Composition`: defines one or more resources and their configuration
-
-In addition to making provisioning available at the namespace scope,
-[composition] also allows for multiple types of managed resources to satisfy the
-same namespace-scoped resource. In the examples below, we will define and
-publish a new `PostgreSQLInstance` resource that only takes a single `storageGB`
-parameter, and specifies that it will create a connection `Secret` with keys for
-`username`, `password`, and `endpoint`. We will then create a `Composition` for
-each provider that can satisfy and be parameterized by a `PostgreSQLInstance`.
-Let's get started!
+# Compose Infrastructure
+
+In the [last section] we learned that Crossplane can be extended by installing
+_providers_, which add support for managed resources. A managed resource is a
+Kubernetes custom resource that offers a high fidelity representation of an
+infrastructure primitive, like an SQL instance or a firewall rule. Crossplane
+goes beyond simply modelling infrastructure primitives as custom resources - it
+enables you to define new custom resources with schemas of your choosing. These
+resources are composed of infrastructure primitives, allowing you to define and
+offer resources that group and abstract infrastructure primitives. We call these
+"composite resources" (XRs).
+
+We use two special Crossplane resources to define and configure new XRs:
+
+- A `CompositeResourceDefinition` (XRD) defines a new kind of composite
+  resource, including its schema.
+- A `Composition` specifies which managed resources a composite resource should
```

Comment:
Can we call out `XMP` here if it will be used as official short name?

Reply:
I don't think there's any consensus that it will be used as the official short name, especially given it's not an acronym.